scripts/deno/iframe-titles.ts

```diff
@@ -0,0 +1,187 @@
+import { array, object, string } from 'https://cdn.skypack.dev/zod@3.17?dts';
+import {
+	DOMParser,
+	Element,
+	Node,
+} from 'https://deno.land/x/deno_dom@v0.1.22-alpha/deno-dom-wasm.ts';
+import { octokit } from './github.ts';
+
+const searchParams = new URLSearchParams({
+	'api-key': 'test',
```

Comment:
Is this key public knowledge already?

Reply:
Yes, it’s listed as an example in https://open-platform.theguardian.com/explore/